js/gatecoin.js

```diff
@@ -341,6 +350,12 @@ module.exports = class gatecoin extends Exchange {
             'side': side,
             'price': trade['price'],
             'amount': trade['quantity'],
+            'cost': trade['price'] * trade['quantity'] + trade['feeAmount'],
```

Comment:
`'feeAmount'` is not present on public trades, I'm afraid...
It has to be under conditionals...

```
let fee = undefined;
let feeCost = this.safeFloat (trade, 'feeAmount');
if (typeof feeCost !== 'undefined') {
    fee = {
        ...
    }
}
```